build_config.rb

```diff
@@ -26,6 +26,7 @@
 
   # ngx_mruby extended class
   conf.gem './mrbgems/ngx_mruby_mrblib'
+  conf.gem :github => 'ksss/mruby-stringio' # required by rack-based-api
```

Comment:
Sure, but I have a question. I'm not familier with mruby build process.
Is ksss/mruby-stringio always linked to ngx_mruby if I add the dependency? Do you prefer it rather than optional linking?